flexdata/data.py

```diff
@@ -493,13 +539,28 @@ def read_flexraymeta(path, sample = 1):
     records['roi'] = roi.tolist()
 
     # Detector pixel is not changed here when binning mode is on...
+    pixel_adjustment = 1
     if (records['mode'] == 'HW2SW1High')|(records['mode'] == 'HW1SW2High'):
         records['det_pixel'] *= 2
         records['img_pixel'] *= 2
+        pixel_adjustment = 2
 
     elif (records['mode'] == 'HW2SW2High'):
         records['det_pixel'] *= 4
         records['img_pixel'] *= 4
+        pixel_adjustment = 4
+
+    if pixel_adjustment != 1:
+       msg = f"Adjusted pixel size by {pixel_adjustement} due to {records['mode']}"
+       logging.info(msg)
+       geom.log(msg)
```

Comment:
`geom` is undefined here and later in this function.

Reply:
Thanks for catching this.